### DISCUSSION TITLE
Should the NLM Media Page be a part of the www.negative-light.com Repo?

### DISCUSSION THREAD

**Discussion by Negative-light** [negative-light-media/.github#9] [Ideas]
This is more of a question thing.
Should we design the Negative Light media website to be a part of the base repo for the [Negative_Light website](https://www.negative-light.com) OR should we make it a completely separate repository with a different URL such as media.negative-light.com?

I would like to hear others thoughts and possible pro's/cons

**Comment by crazysmile11012** [comment]
imo it should

**Reply by Negative-light** [reply]
It should be it's own repo?
